onnxruntime/contrib_ops/cpu/bert/attention_base.cc

```diff
@@ -375,7 +375,7 @@ Tensor* AttentionBase::GetPresent(OpKernelContext* context,
   //   present     : (2, batch_size, num_heads, past_sequence_length + kv_sequence_length, head_size)
 
   past_sequence_length = (nullptr != past) ? static_cast<int>(past->Shape().GetDims()[3]) : 0;
-  std::vector<int64_t> present_dims{2, batch_size, num_heads_, kv_sequence_length + past_sequence_length, head_size};
+  std::array<int64_t, 5> present_dims{2, batch_size, num_heads_, static_cast<int64_t>(kv_sequence_length) + past_sequence_length, head_size};
```

Comment:
Here both kv_sequence_length and past_sequence_length are `int`.  But the result will be converted to int64_t. So we can convert one of them to int64_t first to get rid of the chance of arithmetic overflow.

Reply:
Good practice to use static array.